_includes/pbc/all/install-features/202400.0/install-the-product-offer-service-points-feature.md

```diff
@@ -84,37 +84,37 @@ Make sure that propel entities have been generated successfully by checking thei
-| ProductOfferServiceCollectionResponse  | class | created | src/Generated/Shared/Transfer/ProductOfferServiceCollectionResponseTransfer  |
-| ProductOfferServiceCollectionResponse  | class | created | src/Generated/Shared/Transfer/ProductOfferServiceCollectionResponseTransfer  |
-| ProductOfferServiceCriteria            | class | created | src/Generated/Shared/Transfer/ProductOfferServiceCriteriaTransfer            |
-| ProductOfferServiceConditions          | class | created | src/Generated/Shared/Transfer/ProductOfferServiceConditionsTransfer          |
-| IterableProductOfferServicesCriteria   | class | created | src/Generated/Shared/Transfer/IterableProductOfferServicesCriteriaTransfer   |
-| IterableProductOfferServicesConditions | class | created | src/Generated/Shared/Transfer/IterableProductOfferServicesConditionsTransfer |
-| ProductOfferServiceStorage             | class | created | src/Generated/Shared/Transfer/ProductOfferServiceStorageTransfer             |
-| ProductOfferServiceStorageCollection   | class | created | src/Generated/Shared/Transfer/ProductOfferServiceStorageCollectionTransfer   |
-| ServicePointStorageCollection          | class | created | src/Generated/Shared/Transfer/ServicePointStorageCollectionTransfer          |
-| ServicePointStorageCriteria            | class | created | src/Generated/Shared/Transfer/ServicePointStorageCriteriaTransfer            |
-| ServicePointStorageConditions          | class | created | src/Generated/Shared/Transfer/ServicePointStorageConditionsTransfer          |
-| SynchronizationData                    | class | created | src/Generated/Shared/Transfer/SynchronizationDataTransfer                    |
-| Filter                                 | class | created | src/Generated/Shared/Transfer/FilterTransfer                                 |
+| TRANSFER                                  | TYPE  | EVENT   | PATH                                                                            |
+|-------------------------------------------|-------|---------|---------------------------------------------------------------------------------|
+| ServicePoint                              | class | created | src/Generated/Shared/Transfer/ServicePointTransfer                              |
+| ServicePointCollection                    | class | created | src/Generated/Shared/Transfer/ServicePointCollectionTransfer                    |
+| ServicePointCriteria                      | class | created | src/Generated/Shared/Transfer/ServicePointCriteriaTransfer                      |
+| ServicePointConditions                    | class | created | src/Generated/Shared/Transfer/ServicePointConditionsTransfer                    |
+| ServicePointAddressesBackendApiAttributes | class | created | src/Generated/Shared/Transfer/ServicePointAddressesBackendApiAttributesTransfer |
```

Comment:
Strange, as I see this DTO part of the Service Points feature, please remove it from the connector feature.

Reply:
Removed:
- ServicePointCollection
- ServicePointCriteria
- ServicePointConditions
- ServicePointAddressesBackendApiAttributes